client/auth/client.go

```diff
@@ -1,19 +1,21 @@
 package auth
 
 import (
+	"fmt"
+	"github.com/IBM/satcon-client-go/client/actions"
+	"github.com/IBM/satcon-client-go/client/auth/local"
+	"github.com/IBM/satcon-client-go/client/types"
+	"github.com/IBM/satcon-client-go/client/web"
+	"github.com/dgrijalva/jwt-go"
 	"net/http"
+	"time"
 
 	"github.com/IBM/go-sdk-core/v4/core"
 )
 
-//go:generate go run github.com/maxbrunsfeld/counterfeiter/v6 . AuthClient
-type AuthClient interface {
-	Authenticate(request *http.Request) error
-}
-
```

Comment:
I understand the issue with cyclical dependencies. It's very awkward and confusing IMO to have a package `client/auth` and a type AuthClient which is used in many places, and not to have AuthClient live in `client/auth`. Can you find an alternative solution? I wonder if it would work to have AuthClient and/or your new LocalRazeeClient in a subpackage of `client/auth`?